tests/components/ring/test_sensor.py

```diff
@@ -1,122 +1,46 @@
 """The tests for the Ring sensor platform."""
-from asyncio import run_coroutine_threadsafe
-import unittest
-from unittest.mock import patch
+from .common import setup_platform
 
-import requests_mock
+WIFI_ENABLED = False
 
-from homeassistant.components import ring as base_ring
-import homeassistant.components.ring.sensor as ring
-from homeassistant.helpers.icon import icon_for_battery_level
 
-from tests.common import get_test_home_assistant, load_fixture, mock_storage
-from tests.components.ring.test_init import ATTRIBUTION, VALID_CONFIG
+async def test_sensor(hass, requests_mock):
+    """Test the Ring sensors."""
+    await setup_platform(hass, "sensor")
 
+    front_battery_state = hass.states.get("sensor.front_battery")
+    assert front_battery_state is not None
+    assert front_battery_state.state == "80"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4821145)